application/build.gradle.kts

```diff
@@ -1,22 +1,28 @@
+import java.util.Properties
+
 buildscript {
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath("androidx.navigation:navigation-safe-args-gradle-plugin:2.3.2")
-        classpath("com.google.firebase:firebase-crashlytics-gradle:2.4.1")
+        classpath("androidx.navigation:navigation-safe-args-gradle-plugin:2.3.3")
+        classpath("com.google.firebase:firebase-crashlytics-gradle:2.5.0")
     }
 }
 
+val properties = Properties()
```

Comment:
May be better to move this code block into androidProjectConfiguration? (underhood)
Or, It should lay on top-level gradle build script?

Reply:
Yeah it needs a bit more refinement, next PR will do